libcontainer/container.go

```diff
@@ -74,71 +74,39 @@ type BaseContainer interface {
 	ID() string
 
 	// Returns the current status of the container.
-	//
```

Comment:
This documentation (about which errors can be returned) is removed because
 - libcontainer/error.go already list possible errors;
 - it's not easy to keep these lists up to date.